docs/source/system_reference_guide/jobs_maappy.ipynb

```diff
@@ -135,6 +135,61 @@
+    "```\n",
+    "\n",
+    "Users may provide optional query parameters to filter the list on desired fields. The following example will return a list of jobs submitted by user `mlucas` \n",
+    "that all ran the `job-dps_tutorial_mlucas:main` algorithm:\n",
+    "\n",
+    "```\n",
+    "r = maap.listJobs(\"mlucas\", algo_id=\"job-dps_tutorial_mlucas\", version=\"main\")\n",
+    "```\n",
+    "\n",
+    "Users may also request just the list of jobs id's and their corresponding tags rather than full job details. This is controlled using the `get_job_details` option. By default `get_job_details` is set to `True`, but if set to `False` will return a compact list, as shown in this example:\n",
+    "\n",
+    "```\n",
+    "r = maap.listJobs(\"mlucas\", algo_id=\"job-dps_tutorial_mlucas\", version=\"main\", get_job_details=False)\n",
+    "print(r.text)\n",
+    "```\n",
+    "\n",
+    "```\n",
+    "{\"code\": 200, \"jobs\": [{\"id\": \"e6b6b27d-d409-4f6b-8aa2-505c2dc150fd\", \"tags\": [\"test\"]}, {\"id\": \"fd504b51-096a-42fe-8f1d-a157352dfad0\", \"tags\": [\"test\"]}, {\"id\": \"69dc095a-4895-43a2-951a-763f3a33b0bf\", \"tags\": [\"test\"]}, {\"id\": \"c7983efc-689f-4008-b87a-bc3450213152\", \"tags\": [\"test\"]}, {\"id\": \"41288295-6744-424e-ae26-c1dd9e26c288\", \"tags\": [\"test\", \"test\"]}], \"message\": \"success\"}\n",
+    "```\n",
+    "\n",
+    "These are the available query parameters:\n",
+    "\n",
+    "| Parameter       | Description | Type                        | Optional/Required? | Acceptable Values\n",
+    "|-----------------|-------------|-----------------------------|-------------------|----------|\n",
+    "| algo_id         | algorithm id e.g. `job-dps_tutorial_mlucas` | string  | optional          | Valid string |\n",
+    "| end_time        | job end time   | string e.g. `2024-01-01T00:00:00.000Z`  | optional          | Valid time string |\n",
+    "| get_job_details | Flag that determines whether to return full job details or just a list of job id's with their corresponding tags. Default is `True`.| bool  | optional  | `False` or `True`|\n",
+    "| offset         | pagination offset | number  | optional          | Integer |\n",
```

Comment:
is offset the page-number offset, or the job-number offset? (e.g. if there are 24 jobs with page_size of 20, is page_size only 1 or 2? or 1,2,3…24)?)

Reply:
Offset is the job number offset from the job listing retrieved from Mozart/elasticsearch. So for example, if I make a listJobs request with offset=0 and page_size=2 and get these two job id's:

 0.   03695a4f-86f2-4418-9db0-f55e16a89901
  1.   f3a21979-18fe-4687-88d6-33f18cb5ad05

then I make another request but with offset=1 and page_size=1, I should get this:

f3a21979-18fe-4687-88d6-33f18cb5ad05